### ISSUE TITLE
jupyterlab-0.31.0rc1 starts with a blank page

### ISSUE THREAD

**Issue by stonebig** [jupyterlab/jupyterlab#3477]
when I launch jupyterlab-0.31.0rc1, it opens a blank page:
![jupyterlabindex2](https://user-images.githubusercontent.com/4312421/34324871-11886f8a-e881-11e7-8612-84206e765f49.PNG)

with  jupyterlab-0.30.6 , I had this error
![jupyterlabindex](https://user-images.githubusercontent.com/4312421/34324874-18a6659c-e881-11e7-8152-df86b7cbb22f.PNG)

What may I have done wrong ? (window-10, Python-3.6.4, not anaconda)

**Comment by stonebig** [comment]
in the chrome console, I see this:
````
Uncaught SyntaxError: Unexpected token W in JSON at position 231
    at JSON.parse (<anonymous>)
    at Object.getOption (main.db27b888276f743a5b51.js:1)
    at Object.Js9g (main.db27b888276f743a5b51.js:1)
    at n (manifest.9477600f7a3eb3daa05b.js:1)
    at Object.6 (main.db27b888276f743a5b51.js:1)
    at n (manifest.9477600f7a3eb3daa05b.js:1)
    at window.webpackJsonp (manifest.9477600f7a3eb3daa05b.js:1)
    at main.db27b888276f743a5b51.js:1
````

**Comment by blink1073** [comment]
I found the problem, working on a fix and trying to figure out why it passed the appveyor check.

**Comment by stonebig** [comment]
so I updated jupyterlab-launcher-0.9.1 , answer yes on "build" button, will retry another full build tonight.

... I feel like the Julialang team didn't focus on the right "two langages problem".

**Comment by stonebig** [comment]
ok, apparently solved per jupyterlab-launcher-0.9.1.

thanks a lot, Steven.
